dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -312,6 +312,16 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
+
```

Comment:
just a suggestion here, but we are not leaving spaces when adding new device nodes